docs/install_guides/pypi-openvino-rt.md

```diff
@@ -15,6 +18,11 @@ Before you start the installation, check the supported operating systems and req
 |   macOS* 10.15.x versions                                    | 3.6, 3.7, 3.8, 3.9                                  |
 |   Windows 10*, 64-bit                                        | 3.6, 3.7, 3.8, 3.9                                  |
 
+**C++ libraries** are also required for the installation. To install C++ libraries:
+- On Windows*, you can [download the Visual Studio Redistributable file (.exe)](https://aka.ms/vs/17/release/vc_redist.x64.exe).
+- On macOS*, you can use `xcode-select --install`.
+- On Linux*, you can use `sudo apt install build-essential`.
```

Comment:
really? who said that we need this on OSX and Linux?

Reply:
about Windows, we already state it below "On Windows*, some libraries are necessary to run OpenVINO. To resolve this issue, install the [C++ redistributable (.exe)](https://aka.ms/vs/17/release/vc_redist.x64.exe"

Reply:
> really? who said that we need this on OSX and Linux?

It's suggested by Helena, and Zoya from validation has tested them.

Reply:
> about Windows, we already state it below "On Windows*, some libraries are necessary to run OpenVINO. To resolve this issue, install the [C++ redistributable (.exe)](https://aka.ms/vs/17/release/vc_redist.x64.exe"

Well, it's in the Troubleshooting part, so it's not easy for users to find until they have met issues. It makes more sense to be in System Requirements.

Reply:
> > really? who said that we need this on OSX and Linux?
> 
> It's suggested by Helena, and Zoya from validation has tested them.

I need proofs that w/o these commands python does not work. I've also tested python packages on empty ubuntu 20.04 docker image and I don't build-essentials. If we can think a bit, we can understand that for already **built** package we don't need build tools or build-essential

Reply:
can we remove steps for OSX and Linux?

Reply:
> can we remove steps for OSX and Linux?

Removed. :)